scripts/build.sh

```diff
@@ -639,16 +639,15 @@ if [ "$GAPPS_BRAND" != 'none' ]; then
     echo -e "Extract done\n"
 fi
 
-if [[ "$WSA_MAJOR_VER" -ge 2302 ]]; then
-    echo "Convert vhdx to RAW image"
-    vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/system_ext.vhdx" "$WORK_DIR/wsa/$ARCH/system_ext.img" || abort
-    vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/product.vhdx" "$WORK_DIR/wsa/$ARCH/product.img" || abort
-    vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/system.vhdx" "$WORK_DIR/wsa/$ARCH/system.img" || abort
-    vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/vendor.vhdx" "$WORK_DIR/wsa/$ARCH/vendor.img" || abort
-    echo -e "Convert vhdx to RAW image done\n"
-fi
-
-if [[ "$WSA_MAJOR_VER" -ge 2304 ]]; then
+echo "Convert vhdx to RAW image"
+vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/system_ext.vhdx" "$WORK_DIR/wsa/$ARCH/system_ext.img" || abort
+vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/product.vhdx" "$WORK_DIR/wsa/$ARCH/product.img" || abort
+vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/system.vhdx" "$WORK_DIR/wsa/$ARCH/system.img" || abort
+vhdx_to_raw_img "$WORK_DIR/wsa/$ARCH/vendor.vhdx" "$WORK_DIR/wsa/$ARCH/vendor.img" || abort
+echo -e "Convert vhdx to RAW image done\n"
+
+SYSTEM_IMAGE_TYPE=$(blkid -o value -s TYPE "$WORK_DIR/wsa/$ARCH/system.img")
```

Comment:
I think you should move this command to a seperate func. You can check my code:
https://github.com/YT-Advanced/WSA-Script/blob/096213de2a2ba19e1a6088d1ab132cbd3fd9d567/scripts/build.sh#L134

Reply:
Easier manager, etc